src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -856,26 +864,41 @@ EncodedResults StatefulLLMPipeline::generate(
 
     m_request.infer();
 
-    int64_t last_token = utils::argmax(m_request.get_tensor("logits"), 0);
+    auto logits = m_request.get_tensor("logits");
+    int64_t output_sequence_len = logits.get_shape().at(1);
 
-    results.tokens[0].push_back(last_token);
-    if (streamer_ptr && streamer_ptr->put(last_token)) {
-        return results;
-    }
+    // Swap max_new_token to get_max_new_token()
+    auto sequence_group = std::make_shared<SequenceGroup>(
+        0 /* request_id */, input_ids, config, 1 /* block_size */);
+    sequence_group->update_processed_tokens_num(input_ids.get_size());
```

Comment:
```suggestion
    sequence_group->update_processed_tokens_num(sequence_group->get_prompt_len() - output_sequence_len);
```

Reply:
Will it work w/o SLICE_OUT?

Reply:
I suppose yes.

w/o Slice output len is the same as prompt len and, hence, number of processed tokens is 0

Reply:
I agree, however, for first input_ids we have length == length of prompt, for example, `18`. But `output_sequence_len` is equal to 1024 for first `logits` as it is output from prefill model